src/cli/onefuzz/templates/afl.py

```diff
@@ -106,13 +110,16 @@ def basic(
 
         target_exe_blob_name = helper.target_exe_blob_name(target_exe, setup_dir)
 
-        if fuzzer_setup_container is None:
-            fuzzer_setup_container = Container(
+        if afl_container is None:
+            afl_container = Container(
                 "afl-linux" if helper.platform == OS.linux else "afl-windows"
             )
 
+        # verify the AFL container exists
+        self.onefuzz.containers.get(afl_container)
```

Comment:
It'd be nice to catch the `ValidationError` on failure and emit a specific and useful message to the user.

Reply:
By using `onefuzz.containers.get` to do the verification, we get an Exception of 'invalid container' which comes to the user as:
 
`ERROR:cli:command failed: request did not succeed: HTTP 400 - {"code": 450, "errors": ["invalid container"]}`

The error handler in the SDK should get updated to do something better, but that's outside of the scope for this PR.